platform/administer/templates/versioning.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Versioning Templates
+sidebar_label: Versioning
+sidebar_position: 4
+---
+
+import Flow, { Step } from "@site/src/components/Flow";
+import NavStep from "@site/src/components/NavStep";
+import Button from "@site/src/components/Button";
+import Label from "@site/src/components/Label";
+
+Templates can be versioned. Versioning a template allows you
```

Comment:
Does this need “Templates can be versioned.” I think you could just start with the second sentence.